kratos/tests/cpp_tests/processes/test_replace_elements_and_condition_process.cpp

```diff
@@ -16,92 +16,48 @@
 
-namespace Kratos
+namespace Kratos::Testing
 {
-namespace Testing
-{
-typedef Node<3> NodeType;
 
 /**
 * Checks if the replacement works with triangles (aka 2D geometries)
 */
 KRATOS_TEST_CASE_IN_SUITE(ReplaceElementsAndConditionsProcess1, KratosCoreFastSuite)
 {
     Model current_model;
-    ModelPart& this_model_part = current_model.CreateModelPart("Main");
-
-    Properties::Pointer p_elem_prop = this_model_part.CreateNewProperties(0);
+    ModelPart& r_model_part = current_model.CreateModelPart("Main");
 
-    // First we create the nodes
-    NodeType::Pointer p_node_1 = this_model_part.CreateNewNode(1, 0.0 , 0.0 , 0.0);
-    NodeType::Pointer p_node_2 = this_model_part.CreateNewNode(2, 1.0 , 0.0 , 0.0);
-    NodeType::Pointer p_node_3 = this_model_part.CreateNewNode(3, 1.0 , 1.0 , 0.0);
-    NodeType::Pointer p_node_4 = this_model_part.CreateNewNode(4, 0.0 , 1.0 , 0.0);
-    NodeType::Pointer p_node_5 = this_model_part.CreateNewNode(5, 2.0 , 0.0 , 0.0);
-    NodeType::Pointer p_node_6 = this_model_part.CreateNewNode(6, 2.0 , 1.0 , 0.0);
-
-    // Now we create the "conditions"
-    std::vector<NodeType::Pointer> element_nodes_0 (3);
-    element_nodes_0[0] = p_node_1;
-    element_nodes_0[1] = p_node_2;
-    element_nodes_0[2] = p_node_3;
-    Triangle2D3 <NodeType>::Pointer p_triangle_0 = Kratos::make_shared<Triangle2D3 <NodeType>>( PointerVector<NodeType>{element_nodes_0} );
-
-    std::vector<NodeType::Pointer> element_nodes_1 (3);
-    element_nodes_1[0] = p_node_1;
-    element_nodes_1[1] = p_node_3;
-    element_nodes_1[2] = p_node_4;
-    Triangle2D3 <NodeType>::Pointer p_triangle_1 = Kratos::make_shared<Triangle2D3 <NodeType>>( PointerVector<NodeType>{element_nodes_1} );
-
-    std::vector<NodeType::Pointer> element_nodes_2 (3);
-    element_nodes_2[0] = p_node_2;
-    element_nodes_2[1] = p_node_5;
-    element_nodes_2[2] = p_node_3;
-    Triangle2D3 <NodeType>::Pointer p_triangle_2 = Kratos::make_shared<Triangle2D3 <NodeType>>( PointerVector<NodeType>{element_nodes_2} );
-
-    std::vector<NodeType::Pointer> element_nodes_3 (3);
-    element_nodes_3[0] = p_node_5;
-    element_nodes_3[1] = p_node_6;
-    element_nodes_3[2] = p_node_3;
-    Triangle2D3 <NodeType>::Pointer p_triangle_3 = Kratos::make_shared<Triangle2D3 <NodeType>>( PointerVector<NodeType>{element_nodes_3} );
-
-    Element::Pointer p_elem_0 = Kratos::make_intrusive<Element>(1, p_triangle_0, p_elem_prop);
-    Element::Pointer p_elem_1 = Kratos::make_intrusive<Element>(2, p_triangle_1, p_elem_prop);
-    Element::Pointer p_elem_2 = Kratos::make_intrusive<Element>(3, p_triangle_2, p_elem_prop);
-    Element::Pointer p_elem_3 = Kratos::make_intrusive<Element>(4, p_triangle_3, p_elem_prop);
-    this_model_part.AddElement(p_elem_0);
-    this_model_part.AddElement(p_elem_1);
-    this_model_part.AddElement(p_elem_2);
-    this_model_part.AddElement(p_elem_3);
+    // Generate a simple mesh
+    CppTestsUtilities::CreatePure2DGeometry(r_model_part);
```

Comment:
See my comment about the naming of this method in the other PR. If we aim to use it in any cpp test we should give a more self-descriptive name.

Reply:
Create2DGeometryWithDummyElements?